Student_BCOs/Erin_Bonner_BCO_000531

```diff
@@ -0,0 +1,178 @@
+{
+    "bco_id": "http://biocomputeobject.org/BCO_000531",
+    "checksum": "2c1d54f325747407204599e40e5941e83c4ca2a0ad6ec74944f2ddd7a76c4b64",
+    "bco_spec_version": "1.3.0",
+    "provenance_domain": {
+        "name": "Multi-omics profiling reveals key signaling pathways in ovarian cancer controlled by STAT3.",
+        "version": "1.0.0",
+        "license": "Attribution 4.0 International (CC BY 4.0)",
+        "created": "2019-10-08T21:13:19.536617",
+        "modified": "2019-10-16T18:53:13.739841",
+        "review": [
+            {
+                "status": "in-review",
+                "reviewer_comment": "Reviewed by BioCompute technical assistant, approval pending modification by author",
+                "reviewer": {
+                    "orcid": "https://orcid.org/0000-0002-8824-4637",
+                    "affiliation": "George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "10-15-2019"
+            }
+        ],
+        "contributors": [
+            {
+                "name": "Erin Bonner",
+                "orcid": "https://orcid.org/0000-0002-1299-6330",
+                "affiliation": "The George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "ebonner@gwmail.gwu.edu"
+            },
+            {
+                "name": "Tiangong Lu",
+                "orcid": "",
+                "affiliation": "Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, MI 48109-2800, USA",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Armand Bankhead, III",
+                "orcid": "",
+                "affiliation": "Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109-2800, USA",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Mats Ljungman",
+                "orcid": "https://orcid.org/0000-0003-1553-6695",
+                "affiliation": "Departments of Radiation Oncology, Rogel Cancer Center, University of Michigan Medical School and Department of Environmental Health Sciences, School of Public Health, University of Michigan, Ann Arbor, MI 48109-2800, USA",
+                "contribution": [
+                    "authoredBy"
```

Comment:
In addition to "authoredBy", each publication author should also have "derivedFrom" contribution type (See #112). 
Please add.